modules/services/picom.nix

```
@@ -269,6 +271,21 @@ in {
       '';
     };
 
+    extraWintypes = mkOption {
+      type = types.str;
+      default = "";
+      example = ''
+        tooltip = { 
+          fade = true
+          shadow = true
+        }
```

don't these need semicolons?

```suggestion
        tooltip = {
          fade = true;
          shadow = true;
        };
```

---

They don't, that's not nix - its libconfig, and that's a string, not an attribute set

---

Well, it looks required here: https://github.com/nix-community/home-manager/pull/2242/files#diff-49f03a93e086f1a4ee685f6a0b5900ceb91d82dc655f2002446b6e879e85c760R40-R43

and here: https://github.com/yshui/picom/blob/next/picom.sample.conf#L414

---

Hmm, this seems like a matter of style, but to keep it consistent, I'll add semicolons